verticapy/datasets/loaders.py

```diff
@@ -85,7 +85,6 @@ def load_dataset(
                 query = query.format(f"LOCAL '{path}'")
                 _executeSQL(query, title="Ingesting the data.")
 
-            _executeSQL("COMMIT;", title="Commit.")
```

Comment:
This is not related to the pylint test. I just found the warning message after loading datasets annoying.
"`COPY` automatically commits itself and any current transaction except when loading temporary tables." It doesn't seem that we load any temporary tables here, do we?